File: apps/console/BasketballConsole/BasketballConsole/Models/CosmosDB/Game..cs
namespace NBA.Models.CosmosDB
{
    public class Game
    {
        
        public string Id { get; set; }
        
        public string SeasonId { get; set; }
        
        public string HomeTeamId { get; set; }
        
        public string HomeTeamName { get; set; }
        
        public List<int> HomePlayerIds { get; set; } = [];
        
        public string VisitorTeamId { get; set; }
        
        public string VisitorTeamName { get; set; }
        
        public List<int> VisitorPlayerIds { get; set; } = [];
        
        public DateTime At { get; set; }
    }
}

File: WebSite with CosmosDB/client/nba-statistics/src/models/Game.ts
export class Game {
    id: string;
    homeTeamId: string;
    homeTeamName: string;
    visitorTeamId: string;
    visitorTeamName: string;
    at: Date;

    constructor(id: string, homeTeamId: string, homeTeamName: string, visitorTeamId: string, visitorTeamName: string, at: Date) {
        this.id = id;
        this.homeTeamId = homeTeamId;
        this.homeTeamName = homeTeamName;
        this.visitorTeamId = visitorTeamId;
        this.visitorTeamName = visitorTeamName;
        this.at = at;
    }
}